lib/llog/api.go

```diff
@@ -0,0 +1,201 @@
+package llog
+
+import (
+	"io"
+)
+
+// Level indicates logging level. The higher the value, the less important.
+// Any non-negative integer is always less important than a named constant.
+type Level int
+
+const (
+	// Least important
+	Info Level = -1 - iota
+	Warn
+	// Most important
+	Err
+)
+
+// Set sets this level from string equivalent. s must be "Err",
+// "Warn", "Info", "0", "1", "2", ... or else Set returns an error. Set
+// ignores the case in s, so passing "ERR", "Err", or "err" as s has the same
+// effect.
+func (l *Level) Set(s string) error {
```

Comment:
Agreed, I had added this to enable Level instances to work with the flag package, but clients can easily write their adapter if they need it.

However, I do think the String() method should remain.